test/libsolidity/SolidityScanner.cpp

```diff
@@ -155,6 +155,50 @@ BOOST_AUTO_TEST_CASE(trailing_dot)
 	BOOST_CHECK_EQUAL(scanner.next(), Token::EOS);
 }
 
+BOOST_AUTO_TEST_CASE(leading_underscore_decimal_is_identifier)
+{
+	Scanner scanner(CharStream("var x = _1.2;"));
```

Comment:
Do we want to use ``var`` here? It's merely the scanner alright, but still. Although we'll probably keep ``var`` as keywords for the suggestions, so maybe it's fine. Same in the tests below.

Reply:
Certainly not. This is a left-over from the original PR (original author). I didn't clear all of them yet. Ideally those scanner tests should only tokenize the minimum required tokens.